Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -641,18 +645,34 @@ function QuestieOptions.tabs.general:Initialize()
                     return "Interface\\OptionsFrame\\VoiceChat-Play", 15, 15
                 end,
                 func = function ()
-                    PlaySoundFile(Sounds.QUEST_COMPLETE_SOUND_FILE)
+                    local soundFile = Questie.db.char.questCompleteSoundChoice ~= nil and Questie.db.char.questCompleteSoundChoice or Sounds.QUEST_COMPLETE_SOUND_FILE
+                    PlaySoundFile(soundFile, "Master")
                 end
             },
+            questCompleteSoundChoice = {
+                type = "select",
+                order = 2.13,
+                values = _GetQuestSoundChoices(),
+                sorting = _GetQuestSoundChoicesSort(),
+                style = 'dropdown',
+                name = function() return l10n('Quest Complete Sound Selection') end,
+                desc = function() return l10n('The sound you hear when a quest is completed'); end,
```

Comment:
These translations need to be added to https://github.com/Questie/Questie/blob/master/Localization/Translations/Options/General.lua